src/modules/NetworkController/public/Get-SdnNetworkControllerInfoOffline.ps1

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+function Get-SdnNetworkControllerInfoOffline {
```

Comment:
do we want this exported as it's own stand-alone function, or just have it done as part of `Get-SdnNetworkController` to automatically call this if NC is down?

Reply:
will change it to private. Get-SdnNetworkController will be enough for public